### ISSUE TITLE
Inazuma Eleven 3 Lag Audio/FPS

### ISSUE THREAD

**Issue by InazumaElevenCitra** [citra-emu/citra#5389]
Hello,
Here is my problem on Citra. This problem doesn't just happen to me. The Inazuma Eleven 3 game doesn't seem to be working on Citra. I only do about 20 fps, while in other games I do 50-60fps.
Also the game audio has a lot of latency.

My logs: 
[citra_log.txt](https://github.com/citra-emu/citra/files/4725889/citra_log.txt)

My Processor: AMD Athlon x4 845 Quad Core
RAM: 8 GB HyperX
Graphic card: AMD Radeon(TM) RX 460
And Supply: 500 WATT

I run lots of games like Grand Theft Auto V which is a big game. But on Inazuma Eleven 3 I have a lot of lag.

(I'm French, sorry for my english)
I hope you can resolve this problem soon enough! Thanks U.

**Comment by KingGamingYT** [comment]
You're using an AMD GPU, which might somewhat explain the slowdown. Also, don't compare a 3DS game on Citra to something like GTA V, considering that runs in a completely different environment, is using probably most of your computer's power, and how the PC version of GTA V is meant and optimized for PCs.

**Comment by InazumaElevenCitra** [comment]
Yes I see. But I know other people with nvidia, rtx 2080 etc ... And they also have this problem. I think the game is not optimized on Citra.

**Comment by InazumaElevenCitra** [comment]
You can even try playing this game on Citra you know.
![Capture](https://user-images.githubusercontent.com/66334765/83685857-98ef3d00-a5e9-11ea-82bc-4f7f2f2456d9.PNG)
